enter-chroot.sh

```diff
@@ -23,4 +23,9 @@ then
 	cp "$HOME/.gnupg/trustdb.gpg" "$HOMEWORLD_CHROOT/home/$USER/.gnupg/trustdb.gpg"
 	cp -R "$HOME/.gnupg/private-keys-v1.d/"* "$HOMEWORLD_CHROOT/home/$USER/.gnupg/private-keys-v1.d"
 fi
+if [ -e "$HOME/.ssh" ]
+then
+	mkdir -p "$HOMEWORLD_CHROOT/home/$USER/.ssh/"
+	cp "$HOME/.ssh/id_rsa" "$HOMEWORLD_CHROOT/home/$USER/.ssh/id_rsa"
```

Comment:
What if they use, for example, an ecsda key, or something else?

Also, your commit message is misspelled.